docs/source/reference/projections.rst

```diff
@@ -341,6 +341,22 @@ OSGB
     ax.gridlines()
 
 
+LambertZoneII
+-------------
+
+.. autoclass:: cartopy.crs.LambertZoneII
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    import cartopy.crs as ccrs
+
+    plt.figure(figsize=(3.2687, 3))
+    ax = plt.axes(projection=ccrs.LambertZoneII())
+    ax.coastlines(resolution='10m')
```

Comment:
Actually maybe this one should add `BORDERS` rather than coastlines, so you get the outline of France?

Reply:
Too late - never mind.

Reply:
Sorry, just wanted to get this out the door so continued on! I agree with this though. I think there are a few that would be good to add borders to (if not all?).